src/software/pipeline/main_LdrToHdrMerge.cpp

```diff
@@ -178,7 +182,30 @@ int aliceVision_main(int argc, char** argv)
         }
     }
     std::vector<std::shared_ptr<sfmData::View>> targetViews;
-    hdr::selectTargetViews(targetViews, groupedViews, offsetRefBracketIndex);
+
+    const int middleIndex = usedNbBrackets / 2;
+    const int targetIndex = middleIndex + offsetRefBracketIndex;
+    const bool isOffsetRefBracketIndexValid = (targetIndex >= 0) && (targetIndex < usedNbBrackets);
+
+    const fs::path lumaStatFilepath(fs::path(inputResponsePath).parent_path() / (std::string("luminanceStatistics.txt")));
+
+    if (!fs::is_regular_file(lumaStatFilepath) && !isOffsetRefBracketIndexValid)
+    {
+        ALICEVISION_LOG_ERROR("Unable to open the file " << lumaStatFilepath.string() << " with luminance statistics. This file is needed to select the optimal exposure for the creation of HDR images.");
+        return EXIT_FAILURE;
+    }
+
+    if (workingColorSpace != image::EImageColorSpace::SRGB)
+    {
+        meanTargetedLumaForMerging = std::pow((meanTargetedLumaForMerging + 0.055) / 1.055, 2.2);
+    }
```

Comment:
Need a comment to explain why.
Shouldn't we make a colorspace conversion (considering that the parameter is always in sRGB)?

Reply:
The parameter applies on the luminance, not on the color. I think it's better to consider that the parameter is in sRGB and to remove the gamma if the specified color space is a linear one.